src/renderers/testing/ReactTestRenderer.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactTestRenderer
+ * @flow
```

Comment:
Flow is fine now that we have a better idea how to use it thanks to PRs from @vjeux!